day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,19 @@
+/*
+for (var i =10; i >= 1; i--) {
```

Comment:
It's nice to see you using iteration and conditionals here. One note on this - the grammatical tense changes when the rhyme is down to _2 speckled frogs_ . The following is what should be logged for that case:
```
2 speckled frogs sat on a log eating some most delicious bugs.
One jumped in the pool where its nice and cool, then there **was** 1 speckled **frog**.
```

It's important to pay attention to these details as you continue to practice in preparation for mod 1.

Reply:
ahhhh fixed, thank you!